ui/src/app/models/new/page.tsx

```
@@ -56,7 +53,6 @@ const processModelParams = (requiredParams: ModelParam[], optionalParams: ModelP
       return acc;
     }, {} as Record<string, string>);
 
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

add these disable statements back, so the errors are ignored.